### PR TITLE
Add kubectl config set example

### LINKED ISSUE #398: kubectl config set: please add some examples
kubectl v1.10
/kind documentation
/sig cli

This command's --help message is not useful at all. Can we please have some commonly used tasks listed in EXAMPLES section of the --help message?

```
$ kubectl config set --help
Sets an individual value in a kubeconfig file

PROPERTY _NAME is a dot delimited name where each token represents either an attribute name or a map key.  Map keys may
not contain dots.

PROPERTY _VALUE is the new value you wish to set. Binary fields such as 'certificate-authority-data' expect a base64
encoded string unless the --set-raw-bytes flag is used.

Options:
      --set-raw-bytes=false: When writing a []byte PROPERTY_VALUE, write the given string directly without base64
decoding.

Usage:
  kubectl config set PROPERTY_NAME PROPERTY_VALUE [options]

Use "kubectl options" for a list of global command-line options (applies to all commands).
```

### PR DESCRIPTION
**Pull request by ahmadnurus**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
Add some example for `kubectl config set` command

**Which issue(s) this PR fixes**:

Fixes https://github.com/kubernetes/kubectl/issues/398

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by prksu** [comment]
cc @soltysh @smarterclayton

File: pkg/kubectl/cmd/config/set.go [hunk 1]
```diff
@@ -40,13 +40,28 @@ type setOptions struct {
...
-var setLong = templates.LongDesc(`
+var (
+	setLong = templates.LongDesc(`
 	Sets an individual value in a kubeconfig file
 
 	PROPERTY_NAME is a dot delimited name where each token represents either an attribute name or a map key.  Map keys may not contain dots.
 
 	PROPERTY_VALUE is the new value you wish to set. Binary fields such as 'certificate-authority-data' expect a base64 encoded string unless the --set-raw-bytes flag is used.`)
 
+	setExample = templates.Examples(`
```
Comment (soltysh) [inline pkg/kubectl/cmd/config/set.go]: I'd prefer the examples be more cluster-type independent. Not everyone are using minikube so for those users these examples won't be of much help.

**Review by soltysh — CHANGES_REQUESTED** [review]

File: pkg/kubectl/cmd/config/set.go (hunk 1 above)
Comment (prksu) [reply pkg/kubectl/cmd/config/set.go]: @soltysh updated

**Review by soltysh — APPROVED** [review]
/lgtm
/approve
Thanks :)

**Pull request merged**